src/test/groovy/pl/tomaszdziurko/jvm_bloggers/SpringContextAwareSpecification.groovy

```diff
@@ -1,19 +1,22 @@
 package pl.tomaszdziurko.jvm_bloggers
 
+import javax.transaction.Transactional
+
 import org.springframework.boot.test.SpringApplicationContextLoader
 import org.springframework.test.context.ActiveProfiles
 import org.springframework.test.context.ContextConfiguration
 
-import spock.lang.Specification;
+import spock.lang.Specification
 
 @ContextConfiguration(classes = [JvmBloggersApplication], loader = SpringApplicationContextLoader)
 @ActiveProfiles("test")
+@Transactional
```

Comment:
It's not required to be added explicitly. It's already provided by [`SpringApplicationContextLoader`](http://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/test/SpringApplicationContextLoader.html) in `@ContextConfiguration` class level annotation.